Makefile

```diff
@@ -35,13 +35,30 @@ $(LOCAL_BINARY): $(SOURCES)
 test:
 	env -i PATH=$$PATH GOPATH=$$GOPATH GOROOT=$$GOROOT GOCACHE=$$GOCACHE go test -timeout=120s -v -cover ./ecs-cli/modules/...
 
-
 .PHONY: integ-test
-integ-test:
-	@echo "Building ecs-cli..."
-	./scripts/build_binary.sh ./bin/local
+integ-test: integ-test-build integ-test-run-with-coverage
+
+.PHONY: integ-test-build
+integ-test-build:
+	@echo "Installing dependencies..."
+	go get github.com/wadey/gocovmerge
+	@echo "Building ecs-cli.test..."
+	env -i PATH=$$PATH GOPATH=$$GOPATH GOROOT=$$GOROOT GOCACHE=$$GOCACHE \
+	go test -coverpkg ./ecs-cli/modules/... -c -tags testrunmain -o ./bin/local/ecs-cli.test ./ecs-cli
+
+.PHONY: integ-test-run
+integ-test-run:
 	@echo "Running integration tests..."
-	go test -tags integ -v ./ecs-cli/integ/e2e/...
+	go test -tags integ ./ecs-cli/integ/e2e/...
+
+.PHONY: integ-test-run-with-coverage
+integ-test-run-with-coverage: integ-test-run
```

Comment:
Just for my own understanding... what this is technically doing is just processing the coverage report produced by `integ-test-build`?